bin/run-prober

```diff
@@ -6,4 +6,4 @@ docker pull civiform/civiform-browser-test:latest
 docker run \
   -e BASE_URL="https://staging-aws.civiform.dev" \
   civiform/civiform-browser-test:latest \
-  /bin/bash -c "yarn install && yarn probe"
+  -c "yarn install && yarn probe"
```

Comment:
I don't fully understand now what it does. Who receives "-c" option? Is it a default script which is part of civiform-browser-test?

Reply:
it is pretty confusing, but that's how docker entrypoints work. The entry point is /bin/bash and defined in https://github.com/civiform/civiform/blob/615e7008b1c3e4273ab34556a1e6b692d5b7e5ca/browser-test/playwright.Dockerfile#L27.

Reply:
I see. Can you add coment saying that we are relying on entrypoint from the docker which is bash.